bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -179,10 +170,17 @@ impl RestApiConfigBuilder {
 /// REST API configuration.
 #[derive(Clone)]
 pub struct RestApiConfig {
+    /// REST API binding address.
     pub(crate) binding_socket_addr: SocketAddr,
```

Comment:
I'ld suggest renaming to `bind_socket_addr`.

Reply:
done